### ISSUE TITLE
Updates are not reported into BZ

### ISSUE THREAD

**Issue by voxik** [fedora-infra/the-new-hotness#252]
It appears that BZ reporting is broken. Yesterday, I was notified about these two updates via email:

~~~
A new version of "bindex" has been detected: "0.8.1" newer than "0.8.0", packaged as "rubygem-bindex"
A new version of "bindex" has been detected: "0.8.0" newer than "0.7.0", packaged as "rubygem-bindex"
~~~

But the bindex update bug has not been updated:

https://bugzilla.redhat.com/show_bug.cgi?id=1694121

**Comment by Zlopez** [comment]
Thanks for the report, it looks like the-new-hotness currently experiences issue on production. I will try to fix this ASAP.

**Comment by Zlopez** [comment]
After some digging, I found out, that this is caused on Bugzilla side by this issue - https://bugzilla.redhat.com/show_bug.cgi?id=1679385

After this will be fixed the reporting will work again and thanks to fedora-messaging it will report every update till this stopped working.

**Comment by janvlug** [comment]
I noticed also that the Bugzilla [bug](https://bugzilla.redhat.com/show_bug.cgi?id=1719750) for the availability of a [new nextcloud-client release](https://release-monitoring.org/project/18381/) is not updated.

**Comment by Zlopez** [comment]
@janvlug This is the same issue. I recommend to add to CC of the bugzilla bug above so the developers could see that this issue is affecting more people.

**Comment by Zlopez** [comment]
The bug on bugzilla side was fixed, but there is a new issue that is now blocking the-new-hotness. 
https://pagure.io/fedora-infrastructure/issue/8095

I will try to create some way to handle these issues on bugzilla side more gracefully.

**Comment by Zlopez** [comment]
This is now fixed and I created a #255 to prevent crash when this happens in the future.
